lib/pushmi_pullyu/logging.rb

```diff
@@ -21,14 +21,17 @@ class << self
 
     attr_writer :logger
```

Comment:
I don't have a strong opinion on initializing the logs.  I think what you've done works as well as anything else.

I was trying to find out if Ruby community has a best practice around attribute accessors vs instance variables because I  see here that logger is declared this way.  I couldn't find anything clear.